library/zap.ts

```diff
@@ -88,6 +90,17 @@ export const publishZapReceipt = async (
   const aTag = zapRequestEvent.tags.find((x) => x[0] === "a");
   const pTag = zapRequestEvent.tags.find((x) => x[0] === "p");
 
+  ///////// TEMPORARY /////////
+  try {
+    const btc24Tag = zapRequestEvent.tags.find((x) => x[0] === "btc24");
+    if (btc24Tag) {
+      handleConferenceZap(zapRequestEvent);
+    }
+  } catch (e) {
+    log.error(`Error handling conference zap: ${e}`);
+  }
+  ////////// TEMPORARY /////////
```

Comment:
nit - was thinking we shove all of this into `handleConferenceZap`, but it really doesnt matter either way
```suggestion
  ///////// TEMPORARY /////////
  handleConferenceZap(zapRequestEvent);
  ////////// TEMPORARY /////////
```